examples/sdp3x_manual_address/sdp3x_manual_address.ino

```diff
@@ -10,35 +10,37 @@ SDP3XSensor sdp(SDP3XSensor::SDP3X_I2C_ADDR_21); // Address 0x21
 // SDP3XSensor sdp(SDP3XSensor::SDP3X_I2C_ADDR_23); // Address 0x23
 
 void setup() {
-  Wire.begin();
-  Serial.begin(9600);
-  delay(1000); // let serial console settle
-
-  int ret = sdp.init();
-  if (ret == 0) {
-      Serial.print("init(): success\n");
-  } else {
-      Serial.print("init(): failed, ret = ");
-      Serial.println(ret);
-      while (true) {
-        delay(1000);
-      }
-  }
+  bool error = false;
+  do{
+    Wire.begin();
+    Serial.begin(9600);
+    delay(1000); // let serial console settle
+    int ret = sdp.init();
+    if (ret == 0) {
+          Serial.print("init(): success\n");
+    } else {
+      error = true;
+          Serial.print("init(): failed, ret = ");
+          Serial.println(ret);
+          delay(1000);
+    }
+   
+  }while(error);  
 }
 
 void loop() {
   int ret = sdp.readSample();
   if (ret == 0) {
-      Serial.print("Differential pressure: ");
-      Serial.print(sdp.getDifferentialPressure());
-      Serial.print("Pa | ");
+    Serial.print("Differential pressure: ");
```

Comment:
Although this fixes an inconsistency in the style, it's not a change that belongs in this commit i.e. it has nothing to do with "periodic checking". Please create a separate commit for this style fix. Also, it should be consistently applied to every example file, not just this one.